### PR TITLE
docs(website): portuguese translation

### PR DESCRIPTION
**Pull request by CookieDasora**
## Summary
Partial work of #1065 which translates the following files:
- `index.md`
- `404.md`
- `linter/index.mdx`
- `guides/getting-started.mdx`

## Test Plan

- Check if there is no missing translation in the translated files.
- Check if the translations are correct and if they adhere to the grammatical rules of Portuguese.

### CODE REVIEW

File: website/astro.config.ts [hunk 1]
```diff
@@ -130,6 +130,10 @@ export default defineConfig({
 					label: "简体中文",
 					lang: "zh-CN",
 				},
+				"pt-br": {
+					label: "Português",
+					lang: "pt-br",
```
Comment (victor-teles) [inline website/astro.config.ts:135]:
```suggestion
					lang: "pt-Br",
```

File: website/src/content/docs/pt-br/index.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+head:
+  - tag: title
+    content: Biome
+template: splash
+description: Formate, verifique erros e muito mais em uma fração de segundo.
+hero:
+  title: Uma toolchain para seu projeto web
+  tagline: Formate, verifique erros e muito mais em uma fração de segundo.
+  image:
+    alt: Biome, Toolchain of the web
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/index.mdx:12]: Let's also translate the `alt`

File: website/src/content/docs/pt-br/index.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+title: Biome
+head:
+  - tag: title
+    content: Biome
+template: splash
+description: Formate, verifique erros e muito mais em uma fração de segundo.
+hero:
+  title: Uma toolchain para seu projeto web
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/index.mdx:9]: Suggestion: translate the "toolchain" to "conjunto de ferramentas"

File: website/src/content/docs/pt-br/index.mdx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    Projetado para suportar codebases de qualquer tamanho. Focado no crescimento
+    do projeto, em vez de suas ferramentas.
+  </Card>
+  <Card title="Otimizado" icon="setting">
+    Com uma integração interna eficiente, conseguimos reutilizar trabalhos
+    anteriores, e qualquer melhoria em uma ferramenta beneficia todas elas.
+  </Card>
+  <Card title="Prático e informativo" icon="information">
+    Esqueça aquelas mensagens de erro obscuras! Quando alguma coisa tá errada,
+    nós falamos exatamente o que está errado e como concertar!
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/index.mdx:49]: nit:

```suggestion
    Esqueça aquelas mensagens de erro obscuras! Quando alguma coisa está errada,
    nós falamos exatamente o que está errado e como consertar!
```

File: website/src/content/docs/pt-br/index.mdx (hunk 2 above)
Comment (CookieDasora) [reply website/src/content/docs/pt-br/index.mdx:12]: Currently, I can't think of an appropriate translation for this. Do you have any suggestions?

File: website/src/content/docs/pt-br/index.mdx (hunk 3 above)
Comment (CookieDasora) [reply website/src/content/docs/pt-br/index.mdx]: Implemented suggestion in commit 7ee1d34

File: website/src/content/docs/pt-br/index.mdx (hunk 4 above)
Comment (CookieDasora) [reply website/src/content/docs/pt-br/index.mdx]: Fixed in commit e7a6180!

File: website/src/content/docs/pt-br/guides/getting-started.mdx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+O comando `check` executa múltiplas ferramentas de uma vez. Até o momento, ele:
+
+- formata arquivos
+- analisa arquivos
+- organiza as importações
+
+## Instale um plugin de editor
+
+Recomendamos instalar um plugin de editor para aproveitar ao máximo o Biome. Dê uma olhada na [página de editores](/pt-br/guides/integrate-in-editor) para saber qual suporta Biome.
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/guides/getting-started.mdx:90]:
```suggestion
Recomendamos instalar um plugin de editor para aproveitar ao máximo o Biome. Dê uma olhada na [página de editores](/pt-br/guides/integrate-in-editor) para saber qual oferece suporte ao Biome.
```

File: website/src/content/docs/pt-br/linter/index.mdx [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+## Correções de código
+
+As regras de Lint podem fornecer correções automáticas de código. O Biome classifica as correções em dois tipos.
+
+### Correções seguras
+
+As correções seguras não alteram a semântica do seu código. Elas podem ser aplicadas sem uma análise explicita.
+
+Para aplicar _correções seguras_, utitlize o parâmetro `--apply`:
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/linter/index.mdx:41]:
```suggestion
Para aplicar _correções seguras_, utilize o parâmetro `--apply`:
```

File: website/src/content/docs/pt-br/index.mdx (hunk 2 above)
Comment (victor-teles) [reply website/src/content/docs/pt-br/index.mdx:12]: My suggest is something like "Biome, Conjunto de ferramentas da web".

What do you think?

Comment (CookieDasora) [reply website/src/content/docs/pt-br/index.mdx:12]: LGTM!

File: website/src/content/docs/pt-br/linter/index.mdx [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+Para mais informações sobre todas as opções disponíveis, veja a página sobre [o CLI](/reference/cli#biome-lint)
+
+## Princípios das Regras
+
+Nós acreditamos que as regras devem ser informativas e que expliquem ao usuário porque uma regra é acionada, além de orientar o usuário sobre o que deve ser feito para corrigí-lo. Uma regra deve seguir esses **princípios**:
+
+1. Explicar o erro para o usuário. Geralmente, essa é a mensagem do diagnóstico.
+2. Explicar para o usuário porque o erro foi acionado. Geralmente, isso é implementado com um parâmetro adicional.
+3. Falar para o usuário o que ele precisa fazer. Geralmente, isso é implementado usando um "code action". Se o "code action" não for aplicável uma nota deve dizer o que o usuário precisa fazer para arrumar o erro.
```
Comment (CookieDasora) [inline website/src/content/docs/pt-br/linter/index.mdx:29]: Any suggestions for translating "code action"?

Comment (victor-teles) [reply website/src/content/docs/pt-br/linter/index.mdx:29]: @CookieDasora "Ação no código" sounds good to you?

**Review by victor-teles — APPROVED** [review]
@CookieDasora Thank you for the contribution

Looks good to me :)

**Pull request merged**